packages/corvid-cli/test/clone.e2e.js

```diff
@@ -401,4 +402,34 @@ describe("clone", () => {
       );
     });
   });
+  describe("Failed clone", () => {
+    it("should clean files", async () => {
+      const paths = ["src", ".corvid"];
+      editorServer = await localFakeEditorServer.start();
+      const { tempDir, siteUrl } = await setupSuccessfullClone();
+      process.env.QUERY = "failOnClone=true";
+      fetchMock.mock(
+        `http://frog.wix.com/code?src=39&evid=200&msid=12345678&uuid=testGuid&csi=${
+          process.env.CORVID_SESSION_ID
+        }&status=fail`,
+        JSON.stringify({})
+      );
+
+      const promise = clone({
+        url: siteUrl,
+        dir: tempDir
+      });
+      await expect(promise).rejects.toThrow(
+        getMessage("Pull_Client_Console_Fatal_Error")
+      );
+      await expect(
+        Promise.all(
+          paths.map(relativePath => {
```

Comment:
i'm not sure i understand this part.
why not just readdir from the root path and expect it to be empty (in this case)

Reply:
done